src/other/ThreadedRecoilLoopBase.C

```diff
@@ -148,5 +159,53 @@ ThreadedRecoilLoopBase::operator() (const PKARange & pka_list)
       // done with this recoil
       delete recoil;
     }
+
+    // Process instantaneous recombination of this PKA's defects
+    if (true && !_vacancy_buffer.empty())
```

Comment:
why the `true`? `if (true && !_vacancy_buffer.empty())` and `if (!_vacancy_buffer.empty())` are totally equivalent, are they not?